.github/workflows/go.yml

```diff
@@ -1,32 +1,29 @@
 name: Go
+
 on: [push, pull_request]
-jobs:
 
+jobs:
   build:
     name: Build
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
+        go: ['1.13', '1.x']
```

Comment:
I think that here quotes are not needed. I've noticed that sometimes 1.13.x notation is used. Is 1.13 selecting the latest minor release, or the first one?

Reply:
I removed the quotes. It is selecting latest minor release.